js/tsconfig.json

```
@@ -7,6 +7,7 @@
     "baseUrl": ".",
     "paths": {
       "flarum/*": ["../vendor/flarum/core/js/dist-typings/*"]
-    }
+    },
+    "module": "ES2020"
```

```suggestion
    "module": "es2020"
```

By convention I think we keep this lowercase.

@davwheat I wonder if we should bump our TSConfig to use newer versions of JS for target / module?